test/stdlib/Integer_test.rb

```diff
@@ -137,11 +137,6 @@ def test_abs_abs2
     3.abs2
   end
 
-  def test_digits
-    1.digits
-    1.digits(2)
-  end
```

Comment:
https://github.com/ruby/ruby-signature/blob/e11bce363a1cc67125cfdbd4491ce1512b9aacc0/test/stdlib/Integer_test.rb#L189-L194
Another definition is here.